Sources/Models/NSConstraintLayoutSet.swift

```diff
@@ -24,18 +24,18 @@
 
 import UIKit
 
-public class NSLayoutConstraintSet {
+internal class NSLayoutConstraintSet {
```

Comment:
We just need to add an API breaking CHANGELOG entry under removed for this. Then we can merge

Reply:
👌